### DISCUSSION TITLE
How do I get rid of the text at the bottom of the UI?

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#1754] [Q&A]
![image](https://github.com/vladmandic/automatic/assets/117453319/f0f3ce7d-48db-4ae1-a762-2ad93a793b9f)

**Comment by Tillerz** [comment]
1. automatic/user.css

```
.log-monitor { display:none }
```

or

2. disable "show log view" in `Settings / Live Preview`:
(dont forget to reload the UI after)

![image](https://i.gyazo.com/93c260fad0672d88b1d069e8f86149b7.png)

**Reply by githubUser01946** [reply]
Thanks I will try now :)

**Comment by vladmandic** [comment]
do not modify css when option to show/hide log is already in settings.
